boa/src/vm/instructions.rs

```diff
@@ -57,6 +57,11 @@ pub enum Instruction {
     Neg,
     BitNot,
     Not,
+
+    DefVar(usize),
+    DefLet(usize),
+    DefConst(usize),
+    InitLexical(usize),
```

Comment:
I think these need some documentation on what the associated value is, and what do they do. We should also add documentation to the rest, but these have an extra value with them.

Reply:
I added documentation for the ones in this PR